dockerfiles/ci/xfail_tests/5.4.list

```diff
@@ -0,0 +1,244 @@
+tests/run-test/test005.phpt
```

Comment:
How were these files generated? Do we have a method for keeping them up to date? :)

Reply:
Those were generated after manually looking at the output of failed tests. And comparing them to a baseline. 
E.g. 
SSL tests have failed with extension being turned off.
Some tests failed because `var_dump` output has changed because of new objects in the object stack.

Also yesterday while disabling those I noticed that there are some (quite a few) tests in 7.1 and 7.2 that I missed previously that seem to need more investigation.

About updating them - the end goal I believe should be that we make them all pass (var_dump ones probably for the last). 

Right now they can serve as a baseline and a safeguard against regressions.